### ISSUE TITLE
New quest: What kind of way is this?

### ISSUE THREAD

**Issue by Riksu9000** [streetcomplete/StreetComplete#5840]
### General

Ask to specify the type of way of cycleways lacking defining tags.

There are many highway=cycleway that don't define foot access. While this tagging is valid for a bicycle only path, in my experience it is almost always used incorrectly. Bicycle only paths are usually tagged with foot=no, and ways with only highway=cycleway are often actually combined cycle and footways, paths or crossings/connections. This causes issues for pedestrian navigation.

This might be controversial, because it would mean tagging just highway=cycleway would be considered incorrect or at least insufficient.

Alternatively these ways could be shown in red or another color in the cycleway overlay, but this wouldn't be quite as useful.

Affected tag(s) to be modified/added: foot, bicycle, highway etc.
Question asked: **What kind of way is this?**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort)

### Ideas for implementation

**Element selection:**
highway=cycleway without foot, bicycle or sidewalk. I think StreetComplete always adds at least one of these tags on a cycleway. https://github.com/streetcomplete/StreetComplete/blob/9c567ccea2dc4151121e91c2b749b53d6e7dc04b/app/src/main/java/de/westnordost/streetcomplete/osm/cycleway_separate/SeparateCyclewayCreator.kt

Edit: probably safe to ignore ways tagged with oneway or anything about crossing as well.

**Proposed UI:**
The same form as on the cycleway overlay.

**Comment by matkoniecz** [comment]
"Easily answerable by any pedestrian" - note that at least in Poland I expect that many people would be unable to answer it corrrectly

**Comment by mnalis** [comment]
**TL;DR:**  IMHO sounds problematic for _global_ quest. Perhaps per-country might work after community discussion? SCEE may have user-configurable workaround already.

Details follow:

---

> "Easily answerable by any pedestrian" - note that at least in Poland

I agree; it is problematic in other countries too. 

However, if there is specific country consensus that `highway=cycleway` is inadequate in that country without further access qualifications, and that most pedestrians (even those who have never ridden a bicycle) will for sure know how to answer such quest, that might (or might not) warrant separate quest limited only for that country; but it would need serious discussion with that community.

> in my experience it is almost always used incorrectly

I'd not say *incorrectly*, but ***incompletely***. All that [highway=cycleway](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dcycleway) claims is that it is dedicated for cyclists. i.e. `bicycle=designated`

It makes **no other claims** - e.g. whether it is also allowed (or forbidden) for pedestrians, mofas / mopeds / speed pedelecs etc. (see e.g. https://github.com/streetcomplete/StreetComplete/issues/5565). So yeah, StreetComplete allows to **improve much** OSM data, but it does claim to be able to improve **all** data in all possible cases. 

> highway=cycleway are often actually combined cycle and footways, paths or crossings/connections. This causes issues for pedestrian navigation.

Why? Is some specific (yet unnamed here) pedestrian router assuming that `highway=cycleway` always implies `foot=no` worldwide? If that is the issue, than that is a "bug" (at at least significant inadequacy) at their side, and perhaps you should raise issue with that specific router and point out that unless `foot=no` is explicitly tagged, `highway=cycleway` are (in many, but not all) cases allowed for use by pedestrians. 

You may point them to https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access_restrictions#By_country which shows how full meaning on (otherwise unqualified with specific access tags) defaults for `highway=cycleway` differ in different countries. 

>  Alternatively these ways could be shown in red or another color in the cycleway overlay, but this wouldn't be quite as useful.

As an immediate data-correction measure, if you want, you can try out [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) (StreetComplete _"Expert Edition"_ fork), which has support for that.


<details>

SCEE  has feature called _"Custom Overlay"_ which I believe should  allow you to show those "incomplete"  `highway=cycleway` with different visual outline, allowing you to hunt them down and edit them. 

SCEE also has custom element selection for most (but not all) Quests, but `Is there a bicycle lane or path here? What type?` quest is one of the few which misses that feature at the moment (if you'd like that quests pops up automatically for such "incomplete" cycleways, you may request that "gear :gear:" customization feature be enabled for that quest too at [SCEE issue tracker](https://github.com/Helium314/SCEE/issues).

</details>

**Comment by Riksu9000** [comment]
Ah sorry, I was under the impression that highway=cycleway implies foot=no, and was also debugging a way that had foot=no explicitly!

However, StreetComplete calls a way with only highway=cycleway bicycle only. Is that a bug?

Anyway, I'll close this for now, since it looks like I misunderstood it.

**Comment by mnalis** [comment]
> However, StreetComplete calls a way with only highway=cycleway bicycle only. Is that a bug?

Well, I could be wrong, but I'd guess that this is due to StreetComplete (probably necessary) over-simplifying. 

StreetComplete would need to have computer-readable version of that whole list of [country access defaults for cycleways](https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access_restrictions#By_country), (_**and**_ keep it up-to-date forever), just in order to change that text depending on what country the user is mapping. That would be **a lot** of extra work. So it likely just goes with text for [world-wide default](https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access_restrictions#Worldwide), ignoring that many countries have different defaults :man_shrugging: 

Also if you look at that country list of default, it is not always simple `foot=yes` or `foot=no`; there are often many footnotes which explain the extra restrictions and exceptions, which are so complex that they are not even possible to encode into OSM tags themselves. So even if StreetComplete did invest huge effort to try to implement that, it would _still_ be wrong occasionally. 
World is a complex place, and OSM tagging is trying to simplify it enough for information to be usable; but that necessarily leads to inaccuracies.

**Comment by matkoniecz** [comment]
> However, StreetComplete calls a way with only highway=cycleway bicycle only. Is that a bug?

In cycleway overlay or somewhere else?

**Comment by Riksu9000** [comment]
Yes, in the cycleway overlay. I'm guessing it's because there are no placeholder types, it can only show one of the answers.
